src/config-schema.ts

```diff
@@ -152,6 +152,79 @@ export const configSchema = {
           },
         ],
       },
+      {
+        title: 'Patient Search',
+        description: 'Learn how to search for a specic patient in the system and make them to active visit.',
+        steps: [
+          {
+            target: '[name="SearchPatientIcon"]',
+            title: 'Search Patient',
+            content: 'Click here to search for a patient in the system.',
+            disableBeacon: true,
+            disableOverlayClose: true,
+            spotlightClicks: true,
+            hideNextButton: true,
+            data: {
+              autoNextOn: '[data-testid="patientSearchBar"]',
+            },
+          },
+          {
+            target: '[data-testid="patientSearchBar"]',
+            title: 'Search Bar',
+            content:
+              'This is the search bar. You can search for a patient by entering their name, ID, or other identifiers here.',
+            placement: 'left',
+            disableBeacon: true,
+            disableOverlayClose: true,
+            hideBackButton: true,
+            spotlightClicks: true,
+            hideNextButton: true,
+            data: {
+              autoNextOn: '[data-testid="floatingSearchResultsContainer"]',
+            },
+          },
+          {
+            target: '[data-testid="floatingSearchResultsContainer"]',
+            title: 'Search Results',
+            content: 'Click on the patient that has not visited and you want to search',
+            // spotlightPadding: 40,
+            spotlightClicks: true,
+            placement: 'left',
+            disableBeacon: true,
+            hideBackButton: true,
+            hideNextButton: true,
+            disableOverlay: true,
+            data: {
+              autoNextOn:
+                '.-esm-patient-chart__visit-header__startVisitButton___sPV4T.cds--btn.cds--layout--size-lg.cds--btn--primary',
+            },
+          },
+          {
+            target:
+              '.-esm-patient-chart__visit-header__startVisitButton___sPV4T.cds--btn.cds--layout--size-lg.cds--btn--primary',
+            title: 'Start Visits',
+            content: 'Click here to start a visit for the selected patient.',
+            disableBeacon: true,
```

Comment:
What does this property do?

Reply:
If the beacon is enabled, it will show a red beacon near the target instead of showing the tooltip. (clicking on the beacon opens the tooltip).

https://github.com/user-attachments/assets/9e72a685-88f0-422a-9ba2-08e55bf5fdad